### DISCUSSION TITLE
error when "sudo gedit /etc/profile"

### DISCUSSION THREAD

**Discussion by AimZhang** [peridigm/peridigm#265] [General]
Dear, I am a new beginner at using peridigm. after installing openmpi, I couldn't use 'mpicc', so i want to input the MPI path into '/etc/profile'. But the error will show below:
dbus[45760]: Dynamic session lookup supported but failed: launchd did not provide a socket path, verify that org.freedesktop.dbus-session.plist is loaded!

What should I do to solve it? Thank you very mcuh.

**Comment by sahirbutt** [comment]
Can you give a bit more context, what command did you exactly run to get this error?

**Reply by AimZhang** [reply]
Hello, that error seemed figured out after the upgrade
but when I run 
**sudo gedit /etc/profile**
It will show errors _"2022-09-13 11:09:35.221 gedit[4511:47805] TSM AdjustCapsLockLEDForKeyTransitionHandling - _ISSetPhysicalKeyboardCapsLockLED Inhibit"_
can you help to resolve it?
Thank you very much.

**Reply by johntfoster** [reply]
This is not related to Peridigm at all.  Closing.
